### DISCUSSION TITLE
Add a gallery for themes and plugins on ohmyz.sh

### DISCUSSION THREAD

**Discussion by ThatXliner** [ohmyzsh/ohmyzsh#9706] [Ideas]
On ohmyz.sh, there should be a theme and plugin gallery. Unless, of course, you guys want to keep it in the wiki. I am assuming you guys own the ohmyz.sh domain

**Comment by glc-mvion** [comment]
Maybe this could be achieved with something like [eleventy](https://www.11ty.dev), with a bit of development it would pull the data out of the repo and generate all the content dynamically, we could easily rebuild the OMZ theme library at each release !

I'll try to make some PoC (:
